pkg/engine/masterarmresources.go

```diff
@@ -0,0 +1,156 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+package engine
+
+import (
+	"github.com/Azure/aks-engine/pkg/api"
+	"github.com/Azure/go-autorest/autorest/to"
+)
+
+func createKubernetesMasterResources(cs *api.ContainerService) []interface{} {
```

Comment:
I think we wanna rename this to `createKubernetesMasterResourcesVMSS`, if my read is correct that this is only invoked in a master configuration of availability set scenario.

Reply:
We already have a `createKubernetesMasterResourcesVMSS` method. `createKubernetesMasterResources` is for non-VMSS flows

Reply:
Sorry, my bad. I meant rename to `createKubernetesMasterResourcesVMAS`. More VMAS/VMSS confusion...

Reply:
Got it. Sure, will do